### PR TITLE
[gosrc2cpg] - Additional collection ORM unit test

### PR DESCRIPTION
**Pull request by ankit-privado**
Added unit test for `Echo`, `Chi`, `Fiber`, `Buffalo` framework

### CODE REVIEW

File: src/test/scala/ai/privado/languageEngine/go/tagger/collection/CollectionTaggerTest.scala [hunk 1]
```diff
@@ -91,3 +91,198 @@ class CollectionRestTaggerTest extends CollectionTaggerTestBase {
...
+  }
+
+  "CollectionTagger" should {
+    "Test correct collection tagging" in {
+      cpg.method("allArticles").tag.nameExact("COLLECTION_METHOD_ENDPOINT").head.value shouldBe "\"/articles\""
+    }
+  }
+}
+
+class CollectionTaggerGoBuffaloFrameworkTest extends CollectionTaggerTestBase {
```
Comment (pandurangpatil) [inline src/test/scala/ai/privado/languageEngine/go/tagger/collection/CollectionTaggerTest.scala:235]: @ankit-privado Why we have created a class for each unit test?

Comment (ankit-privado) [reply src/test/scala/ai/privado/languageEngine/go/tagger/collection/CollectionTaggerTest.scala:235]: @pandurangpatil For every ORM framework, I have created its own test class.

Comment (pandurangpatil) [reply src/test/scala/ai/privado/languageEngine/go/tagger/collection/CollectionTaggerTest.scala:235]: I think we can merge them into one class. But looks like the `private def getContent()` and `goFileContentMap` variables need to be defined separately. 

For now I am approving it. We have to work on a proper solution and merge all of these unit tests inside one class.

**Review by pandurangpatil — APPROVED** [review]
looks good to me.

**Pull request merged**